### PR TITLE
ECOPD-2000 added helloWorld method

### PR DESCRIPTION
**Pull request by nileshthakare10**

### CODE REVIEW

File: src/main/java/com/javabrains/PracticeREST/UserEntity.java [hunk 1]
```diff
@@ -7,11 +7,16 @@ public static void main(String[] args) {
...
         }
 
         String dependsOnIsTrue = isTrue ? "nice" :"bad ";
-        System.out.println(dependsOnIsTrue);
 
+
+    }
+
+    void helloWorld(){
+
```
Comment (Mannyp-design) [inline src/main/java/com/javabrains/PracticeREST/UserEntity.java:19]: Nilesh add access modifier public tto improve readability

File: src/main/java/com/javabrains/PracticeREST/UserEntity.java [hunk 2]
```diff
@@ -7,11 +7,16 @@ public static void main(String[] args) {
         boolean isTrue;
 
         if(1>0){
-            isTrue=false;
+            isTrue=true;
         }
 
         String dependsOnIsTrue = isTrue ? "nice" :"bad ";
-        System.out.println(dependsOnIsTrue);
```
Comment (Mannyp-design) [inline src/main/java/com/javabrains/PracticeREST/UserEntity.java:14]: Do not remove this

Comment (nileshthakare10) [reply src/main/java/com/javabrains/PracticeREST/UserEntity.java:14]: Reverted back this line

File: src/main/java/com/javabrains/PracticeREST/UserEntity.java (hunk 1 above)
Comment (nileshthakare10) [reply src/main/java/com/javabrains/PracticeREST/UserEntity.java:19]: Added public access modifier for the corresponding method

**Review by Mannyp-design — APPROVED** [review]

**Comment by Mannyp-design** [comment]
Looks good

**Pull request merged**